scripts/build/autotools/libmongoc/FindDependencies.m4

```diff
@@ -23,5 +23,16 @@ AC_CHECK_TYPE([socklen_t],
               [AC_SUBST(MONGOC_HAVE_SOCKLEN, 0)],
               [#include <sys/socket.h>])
 
-dnl libbson already requires pthreads, so skip a redundant, optional check here
-dnl AX_PTHREAD
+# Check for pthreads. libmongoc's original FindDependencies.m4 script did not
+# require pthreads, but it does appear to be necessary on non-Windows platforms
+# based on mongoc-openssl.c and mongoc-thread-private.h.
```

Comment:
I'm not sure why libmongoc always considered pthreads optional, but they are certainly assumed to exist in these files. It's possible that this was an edge case and the dependency was satisfied by also linking libbson successfully.

In any event, I think this is correct and I'm not going to both libmongoc about it since the M4 file in question no longer exists in their repository.